case_studies/dc2_cataloging/DC2_exp_record.md

```diff
@@ -360,6 +360,26 @@ processed_raw_images = F.relu(raw_images - self.asinh_thresholds_tensor) * self.
 
 3. run dynamic asinh
 
-4. run dynamic asinh
+### 06-15
 
-5. run dynamic asinh
+1. run dynamic asinh
+
+2. run dynamic asinh
+
+3. run fixed thresholds asinh
+
+4. run fixed thresholds asinh
+
+5. run fixed thresholds asinh
+
+### 06-16
+
+1. run fixed thresholds asinh
+
+2. run fixed thresholds asinh
+
+3. run moving avg thresholds asinh
+
+4. run moving avg thresholds asinh
+
+5. run moving avg thresholds asinh

```

Comment:
Ideally hydra would log all the settings used for each run in the correct output directory (the `hparams.yaml` file). I find it hard to keep track of which run is which otherwise, although creating a logging file like this one seems helpful too

Reply:
sometimes, i temporarily change the code and run the experiment, so i use this file to also record the code changes